.github/dependabot.yml

```diff
@@ -28,7 +28,7 @@ updates:
       generic:
         patterns:
           - "external/win-c"
-          - "external/Catch2"
+          - "external/usersim/external/Catch2"
```

Comment:
This line should be deleted since dependabot inside the usersim repo will handle it